src/components/map/map.tsx

```diff
@@ -1,8 +1,66 @@
-import { IonContent } from '@ionic/react';
-import React from 'react';
+import React, { Component } from 'react';
+import { MapContainer, TileLayer, Marker, Popup, MapConsumer } from 'react-leaflet';
+// import 'leaflet/dist/leaflet.css';
+import markerIconPng from "leaflet/dist/images/marker-icon.png";
+import { Icon } from 'leaflet';
 
-const map: React.FC = () => (
-    <span>Map</span>
-);
 
-export default map;
\ No newline at end of file
+interface State {
```

Comment:
Let's find a less generic name for this interface - `MapElements` perhaps (if it's likely to contain ways as well as points)?